bin/worker.js

```diff
@@ -136,8 +136,8 @@ co(function *() {
     config[field] = program[field];
   });
 
-  // If isolated containers is set override capacity...
-  if (config.isolatedContainers) {
+  // If isolated containers is set override capacity (as long as capacity is > 0
+  if (config.isolatedContainers && config.capacity > 0) {
```

Comment:
if for some reason the host config sets capacity to zero, dont' override it with the isolated containers option